### PR TITLE
Add support for riscv64

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Review by superm1 — COMMENTED** [review]
There's more to it than this.  Take a look at all the stuff that was being done on the linked branch on
https://github.com/fwupd/fwupd-efi/issues/19

**Comment by superm1** [comment]
CC @xypron

**Comment by superm1** [comment]
And I think we need at least a cross compile CI job for some OS.

**Comment by hughsie** [comment]
Ohh I was just proxying a downstream patch. Input from @xypron certainly very welcome.

File: efi/meson.build [hunk 1]
```diff
@@ -93,7 +93,7 @@ endif
 
 # older objcopy for Aarch64 and ARM32 are not EFI capable.
```
Comment (CoelacanthusHex) [inline efi/meson.build:94]:
```suggestion
# older objcopy for Aarch64, ARM32 and riscv are not EFI capable.
```
Modify comment at same time.

**Review by CoelacanthusHex — COMMENTED** [review]
I tested it in Arch RISC-V package. It works well.

**Comment by xypron** [comment]
Am 23. April 2023 12:50:27 MESZ schrieb Coelacanthus ***@***.***>:
>@CoelacanthusHex commented on this pull request.
>
>I tested it in Arch RISC-V package. It works well.
>
>> @@ -93,7 +93,7 @@ endif
> 
> # older objcopy for Aarch64 and ARM32 are not EFI capable.
>
>```suggestion
># older objcopy for Aarch64, ARM32 and riscv are not EFI capable.

%s/riscv/RISC-V/

>```
>Modify comment at same time.
>

**Comment by CoelacanthusHex** [comment]
> Am 23. April 2023 12:50:27 MESZ schrieb Coelacanthus ***@***.***>:
> @CoelacanthusHex commented on this pull request. I tested it in Arch RISC-V package. It works well. > @@ -93,7 +93,7 @@ endif # older objcopy for Aarch64 and ARM32 are not EFI capable. ```suggestion # older objcopy for Aarch64, ARM32 and riscv are not EFI capable.
> %s/riscv/RISC-V/
> […](#)
> ``` Modify comment at same time.

Updated. Thanks.

**Review by gmbr3 — CHANGES_REQUESTED** [review]
✓ Builds correctly on openSUSE riscv64
⚠ No riscv64 crt0/lds (SBAT patched) in-tree, useful for people that don't patch gnu-efi

**Pull request closed without merging**